graphene/relay/connection.py

```diff
@@ -107,20 +112,28 @@ def type(self):
                 "Read more: https://github.com/graphql-python/graphene/blob/v2.0.0/UPGRADE-v2.0.md#node-connections"
             )
 
-        assert issubclass(connection_type, Connection), (
-            '{} type have to be a subclass of Connection. Received "{}".'
-        ).format(self.__class__.__name__, connection_type)
+        if not issubclass(connection_type, Connection):
+            raise AssertionError(
+                '{} type have to be a subclass of Connection. Received "{}".'.format(
+                    self.__class__.__name__, connection_type
+                )
+            )
         return type
 
     @classmethod
     def resolve_connection(cls, connection_type, args, resolved):
         if isinstance(resolved, connection_type):
             return resolved
 
-        assert isinstance(resolved, Iterable), (
-            "Resolved value from the connection field have to be iterable or instance of {}. "
-            'Received "{}"'
-        ).format(connection_type, resolved)
+        if not isinstance(resolved, Iterable):
```

Comment:
Rather than repeating the construction:
```
if BOOLEAN_EXPRESSION:
    raise AssertionError("...")
```
Could you DRY things up by writing a helper function that does this and use it across the code as a straight replacement for the `assert` statements?

Reply:
I'll do that!

Reply:
I am trying to do something like this -